docs/examples/lisflood.ipynb

```diff
@@ -5,131 +5,178 @@
    "id": "c02a5bfe",
```

Comment:
<ol><li>noetbook -&gt; notebook</li><li>italic 1990 -&gt; surround file name with backticks</li><li>Why did you make a copy of forcing files why not use them directly, similar to the parameters?</li><li>Can we describe how these forcing files where made for example: </li></ol><p> </p><p>Forcing files on Cartesius where made by running ESMValTool recipe and lisvap in the past . We can now use those files to run the Lisflood model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<ol><li>it is fixed. </li><li>it is fixed.</li><li>we don't have lisvap files in comparison directory and different directories in forcing.load results in error in model.initialize later.</li><li>it is added.</li></ol>

Reply:
3.  So, in ewatercycle, something like this:

```python
forcing = ewatercycle.forcing.load_foreign(target_model='lisflood',
                                           directory='/projects/0/wtrcycle/comparison/lisflood/forcing/',
                                           start_time='1990-01-01T00:00:00Z',
                                           end_time='1990-12-31T00:00:00Z',
                                           forcing_info={
                                               'PrefixPrecipitation': 'lisflood_ERA-Interim_pr_1990_1990.nc',
                                               'PrefixTavg': 'lisflood_ERA-Interim_tas_1990_1990.nc',
                                               'PrefixE0': '/scratch/shared/ewatercycle/lisflood_example/lisflood_forcing_data/lisflood_e0_1990_1990.nc',
                                               'PrefixES0': '/scratch/shared/ewatercycle/lisflood_example/lisflood_forcing_data/lisflood_es0_1990_1990.nc',
                                               'PrefixET0': '/scratch/shared/ewatercycle/lisflood_example/lisflood_forcing_data/lisflood_et0_1990_1990.nc',
                                           }
                                          )
```

does not work. In comparison, we could have different directories for esmvaltool output and lisvap output files. I think because we had used the absolute paths in the config file before.

Reply:
ah, yep, lisvap output files will not be mounted inside containers. OK, then a copy is what we need until #96 is completed.